common/src/main/java/tachyon/collections/PrefixList.java

```diff
@@ -51,10 +62,21 @@ public PrefixList(String prefixes, String separator) {
     }
   }
 
+  /**
+   * Gets the list of prefixes.
+   *
+   * @return the list of prefixes
+   */
   public List<String> getList() {
     return ImmutableList.copyOf(mInnerList);
   }
 
+  /**
+   * Checks if a path is in the list.
```

Comment:
This method is sort of oddly named since it sounds like it should be checking whether `mInnerList.contains(path)`. Can we change the doc to specify that it's checking whether a prefix of `path` is in the prefix list?